readme.md

```diff
@@ -1,23 +1,76 @@
+   1. Subsequent requirements on KPs in this document will specify their application to ReasonerAPI, SmartAPI, and/or KGX interfaces.
+1. Entities in any ReasonerAPI message (ARS/ARA or ARA/KP) or KGX file-based communication are represented using compact URIs (CURIES), which must be expandable to full IRIs using a biolink-model provided json-ld context file.  Entities returned from a non-ReasonerAPI smartAPI-registered KP must provide sufficient information in the registry to allow an automated conversion of the entity identifier to a biolink-model CURIE.
+1. Node Identifiers
+   1. KPs must expose machine readable information about the types of node identifiers that they consume and produce.
+   1. ARAs or other integration tools such as KGX will perform node identifier equivalence translations.
+   1. The consortium will produce or adopt equivalent id sets, which will be shared across Translator tools.  Multiple Translator teams will contribute expertise to these sets, but that expertise will produce centralized results.
+   1. SRI will provide tools for disseminating these equivalent identifiers, drawing on the prior work of multiple Translator teams.
+1. Edge Predicates
+   1. Relationships between entities (edges) have a predicate indicating the specific type of relationship between the entities.
+   1. The biolink model will contain a set of predicates (_biolink predicates_) used to bridge across pre-existing predicate vocabularies
+   1. The biolink model will designate a set of such vocabularies that can be mapped to biolink predicates. These vocabularies are called _biolink-mapped_.
+   1. Predicates in ReasonerAPI messages and KGX files must be biolink predicates.
+   1. Responses from non-ReasonerAPI smartAPI-registered KP must provide sufficient information via the registry that clients can determine the predicate as an identifier from a biolink-mapped vocabulary.
+   1. As a best practice, KPs should map ingested predicates to a biolink-mapped vocabulary as precisely as possible, and rely on tools to convert these predicates into biolink predicates.
+   1. The SRI will provide mapping tools to perform this conversion.
+1. ARAs and KPs may both score answers (provide scores in the message); ARAs are required to score answers.
+1. KPs should not call other KPs.
+1. KPs that implement the Translator Reasoner API must perform the following kinds of reasoning in answering queries:
+   1. Making identifiers more specific, e.g. responding to a query involving an entity with information related to a subclass of that entity.
+   1. Making categories in a query more specific. e.g. responding to a query for a biolink:NamedThing with a particular biolink:ChemicalSubstance.
+   1. Making predicates more specific, e.g. responding to a query for “affects expression of” with an edge with predicate “increases expression of”.
+   1. Inverting predicates. e.g. responding to a query with predicate P with an edge whose predicate is the inverse of P.  
```

Comment:
I think we should also strike this line, as it is no longer relevant.

Reply:
(That is the line re: Inverting predicates)

Reply:
Do you mean item 1?

I confess I'm puzzled why it would be deleted. I recall that we discussed and I thought we can down on the side that resources MUST support either direction when inverses exist, but always return the result using the non-inverse direction? In any case, instead of deleting principle 1, shouldn't we replace it with the final decision of the group?